tiatoolbox/models/classification/patch_predictor.py

```diff
@@ -152,11 +152,9 @@ class CNNPatchPredictor:
         >>> data = [img1, img2]
         >>> predictor = CNNPatchPredictor(pretrained_model="resnet18-kather100k")
         >>> output = predictor.predict(data, mode='patch')
-
         >>> data = np.array([img1, img2])
         >>> predictor = CNNPatchPredictor(pretrained_model="resnet18-kather100k")
         >>> output = predictor.predict(data, mode='patch')
-
```

Comment:
keep the separating line.

Reply:
This is stale comment so i will just resolve this. This should have been addressed in improve model doc PR